runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeSearchPanel.java

```diff
@@ -82,6 +82,8 @@
 
 	private final List<GrandExchangeItems> itemsList = new ArrayList<>();
 
+	private String lookup;
```

Comment:
The scope of this variable shouldn't change.

Reply:
Is there a way you recommend getting around not changing it? Because if I tried to change the value of lookup after initializing it in the method I was getting an error saying that lookup should be a constant on line 190

Reply:
`String lookup = method(searchBar.getText());`